### PR TITLE
Create universite-nangui-abrogoua-ufr-sn

### PR DESCRIPTION
**Pull request by EhoumanEvans**
This a new style for university of Nangui Abrogoua Abidjan Côte d'Ivoire, Département of Sciences of Nature.

### CODE REVIEW

File: universite-nangui-abrogoua-ufr-sn.csl [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+    <category field="botany"/>
+    <category field="science"/>
+    <category field="zoology"/>
+    <updated>2020-04-07T09:57:47+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="and">et</term>
+      <term name="et-al">et al.</term>
```
Comment (EhoumanEvans) [inline universite-nangui-abrogoua-ufr-sn.csl:24]: add here changes

Comment (EhoumanEvans) [reply universite-nangui-abrogoua-ufr-sn.csl:24]: I would like to change make et al. appear in italic in text citation

**Comment by EhoumanEvans** [comment]
Need help to change  et autres (and others) to _et al._
is there anyone to help me?
Thank you

**Comment by EhoumanEvans** [comment]
> Need help to change et autres (and others) to _et al._
> is there anyone to help me?
> Thank you
> 
> Okay done!

oak resolved

**Comment by EhoumanEvans** [comment]
How could I edit the code so that I could have **Fenner M.,** instead of **Fenner, M.,** ?

Thank you

**Comment by EhoumanEvans** [comment]
line added to have (,) after container, journal title

**Comment by EhoumanEvans** [comment]
check line 61 related to name delimiter

**Comment by rmzelle** [comment]
@EhoumanEvans, I see you have a description of the desired style format at https://github.com/EhoumanEvans/CitationStyle/blob/master/universite_nangui_abrogoua_ufr_sn_citation_style.docx, which seems like a document you put together yourself.

Does the UFR Sciences de la Nature of the Université Nagui Abrogoua have an official style guide we could link to? What did you base your examples on?

It also looks like you're using this pull request to make a lot of tweaks to the style. For active development, you're probably better off using a more dedicated tool for that like Zotero's Style Editor (https://www.zotero.org/support/dev/citation_styles/reference_test_pane), https://editor.citationstyles.org/codeEditor/, or https://editor.citationstyles.org/visualEditor/. Those give you quicker feedback about the changes you're making.

**Comment by EhoumanEvans** [comment]
> @EhoumanEvans, I see you have a description of the desired style format at https://github.com/EhoumanEvans/CitationStyle/blob/master/universite_nangui_abrogoua_ufr_sn_citation_style.docx, which seems like a document you put together yourself.
> 
> Does the UFR Sciences de la Nature of the Université Nagui Abrogoua have an official style guide we could link to? What did you base your examples on?
> 
> It also looks like you're using this pull request to make a lot of tweaks to the style. For active development, you're probably better off using a more dedicated tool for that like Zotero's Style Editor (https://www.zotero.org/support/dev/citation_styles/reference_test_pane), https://editor.citationstyles.org/codeEditor/, or https://editor.citationstyles.org/visualEditor/. Those give you quicker feedback about the changes you're making.

Dear @rmzelle thank you for your comment. For the time being the UFR Sciences de la Nature of the Université Nagui Abrogoua has not an official online guide. I am working on the project to make a proposal to the Dean of the UFR. 
Then I am a bit new on GitHub coding and even in coding. Thank you very much for the link of zoetrope offline editor. 

**Please I have a question. I have an old version wish succeeded the verification test, then I made several changes which then failed. I would like to activate the succeeding version and cancel the ones which failed, please could you give me a hand ?**

I want to activate the version with verification number 
5b02c79

Thank you

**Comment by EhoumanEvans** [comment]
I am trying to activate an old commit. The a1ffd97

**Comment by rmzelle** [comment]
@EhoumanEvans, the primary problem at the moment is that you removed the ".csl" file name extension from your style.

**Comment by EhoumanEvans** [comment]
> .csl

Ooops thank you. 
I have just done some changes. Thank you for Zotero style editor it is awesome

**Comment by EhoumanEvans** [comment]
Hooorayyyyyyyyyy
Thank you @rmzelle  for your help. Was very useful. I ll get in touch with the dean of ly dept to have more details. When this style could be available for all?.
Thank you

**Comment by rmzelle** [comment]
> For the time being the UFR Sciences de la Nature of the Université Nagui Abrogoua has not an official online guide.

Is there an offline style guide? Even being able to specify what official (printed) document you based your style and documentation on would be helpful.

> When this style could be available for all?.

Once we address the question above we should be able to accept the style in a few days.

**Comment by adam3smith** [comment]
@EhoumanEvans do you have a reply to @rmzelle 's question above? We'd love to merge this, but would really like some sense of documentation

**Comment by EhoumanEvans** [comment]
> @EhoumanEvans do you have a reply to @rmzelle 's question above? We'd love to merge this, but would really like some sense of documentation

yes, I am working on it. Sorry for the delay.

Kind regards, 
Evans, E.

**Comment by EhoumanEvans** [comment]
> > For the time being the UFR Sciences de la Nature of the Université Nagui Abrogoua has not an official online guide.
> 
> Is there an offline style guide? Even being able to specify what official (printed) document you based your style and documentation on would be helpful.
> 
> > When this style could be available for all?.
> 
> Once we address the question above we should be able to accept the style in a few days.

I have added a .pdf reference guide to justify the citation style.

**Comment by EhoumanEvans** [comment]
A .pdf reference guide file added explaining the citation style. It's based on a APA style style modification.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**